template.yaml

```diff
@@ -16,6 +16,7 @@ Parameters:
       - staging
       - integration
       - production
+    Default: dev
```

Comment:
Any reason why this has been added? Don't think we had a need for a default before, because Github actions sort that out for us.

Reply:
probably a result of the merge of ed's branch `origin/feature/powertools-logging`. will remove it